src/components/AuthenticatedAppHeader/IndividualsButton.jsx

```diff
@@ -38,178 +23,83 @@ export default function IndividualsButton() {
     error,
   } = useIndividualTermQuery(searchTerm);
 
-  useOnEnter(() => {
-    if (inputContent !== '') setSearchTerm(inputContent);
-  });
-
-  const handleClose = () => {
-    setAnchorEl(null);
-  };
-
-  const open = Boolean(anchorEl);
-  const id = open ? 'simple-popover' : undefined;
-
   const resultsCurrent = inputContent === searchTerm;
-  const noResults = searchResults && searchResults.length === 0;
+  const noResults = searchResults?.length === 0;
```

Comment:
If searchResults is nullish, noResults will be false. Don't you want it to be true in this case?

Reply:
i only want it to be true if searchResults are available and there are none. i want false if it's loading or in an error state